docs/types/value-types.rst

```diff
@@ -587,11 +587,14 @@ Enums cannot have more than 256 members.
 The data representation is the same as for enums in C: The options are represented by
 subsequent unsigned integer values starting from ``0``.
 
+Using ``type(NameOfEnum).min`` and ``type(NameOfEnum).max`` you can get the
+smallest and respectively largest value of the given enum.
+
 
 .. code-block:: solidity
 
     // SPDX-License-Identifier: GPL-3.0
-    pragma solidity >=0.4.16 <0.9.0;
+    pragma solidity ^0.8.8 <0.9.0;
```

Comment:
I think `^0.8.8` and `>=0.8.8 <0.9.0` should be equivalent? (with the exception of prereleases)

Reply:
Especially with `^` the `<` is redundant.